### PR TITLE
Sumy web summarizer

### PR DESCRIPTION
**Pull request by LeonardPuettmannKern**
PR checklist:
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by creator on refinery
- [x] Tested by reviewer on localhost:8000/docs
- [x] Tested by reviewer on refinery
- [x] (If added) common code tested in notebook/ script
- [x] Conforms with the agreed upon code pattern

Dependency not yet installed in refinery. Can be tested in the code IDE with this: 

```python
import os
os.system("pip install sumy")

from sumy.parsers.html import HtmlParser
from sumy.nlp.tokenizers import Tokenizer
from sumy.summarizers.lsa import LsaSummarizer as Summarizer
from sumy.nlp.stemmers import Stemmer
from sumy.utils import get_stop_words

ATTRIBUTE: str = "url"
LANGUAGE: str = "english"
SENTENCE_COUNT: int = 5

def sumy_website_summarizer(record):
    parser = HtmlParser.from_url(record[ATTRIBUTE], Tokenizer(LANGUAGE)) 
    stemmer = Stemmer(LANGUAGE)
    summarizer = Summarizer(stemmer)
    summarizer.stop_words = get_stop_words(LANGUAGE)
    summary = summarizer(parser.document, SENTENCE_COUNT)
    return " ".join([str(sentence) for sentence in summary])

print(sumy_website_summarizer(record))
```

### CODE REVIEW

**Comment by LeonardPuettmannKern** [comment]
Implements #284

File: classifiers/_template/_template_func/config.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-from util.configs import build_classifier_function_config
+from util.configs import build_extractor_function_config
```
Comment (DerKernigeFeuerpfeil) [inline classifiers/_template/_template_func/config.py:1]: is this the correct one? shouldnt it be classifier?

File: generators/summarization/sumy_website_summarizer/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This brick module is using the sumy library from GitHub to summarize texts that were scraped from websites like wikipedia or blog articles. The function excepts an url to a website and not a text itself. It uses Latent Semantic Analysis to extract structures of the words and texts used to find relevant sentences. You can read more about the sumy libraray here: https://github.com/miso-belica/sumy
```
Comment (DerKernigeFeuerpfeil) [inline generators/summarization/sumy_website_summarizer/README.md:1]: I think "that were scraped from websites" implies that this process has already been done by sumy, but you query the website first, right? sumy does not work offline, correct?

**Review by DerKernigeFeuerpfeil — CHANGES_REQUESTED** [review]
minor adjustments required

File: generators/summarization/sumy_website_summarizer/README.md (hunk 2 above)
Comment (LeonardPuettmannKern) [reply generators/summarization/sumy_website_summarizer/README.md:1]: Correct!

Comment (LeonardPuettmannKern) [reply generators/summarization/sumy_website_summarizer/README.md]: I now added a sentence that the articles itself are scraped by sumy and that an internet connection is needed.

File: classifiers/_template/_template_func/config.py (hunk 1 above)
Comment (LeonardPuettmannKern) [reply classifiers/_template/_template_func/config.py]: Yes! Fixed it.

**Pull request merged**